### ISSUE TITLE
version 1.12 squeezenet with fully connected layer  Failed to execute on BatchNormalization1d

### ISSUE THREAD

**Issue by momandai** [OAID/Tengine#216]
system: x86 ubuntu18.04
tengine version: 1.12
create squeezenet with fc layer in pytorch, co

- [x]  create squeezenet with fc layer in pytorch
- [x]  convert pth to onnx,and simplify onnx
- [x]  convert simlified onnx to tm
- [ ]  run tm model with build/tests/bin/test_tm failed
`./tests/bin/test_tm /home/test/whale/repo/whcode/cv/whalemgs/onnx_tools/whale_reid_squeezenet_fc_sim.tm 256 128`
![image](https://user-images.githubusercontent.com/24688004/76833211-e4edd980-6865-11ea-991a-99f4958e3e13.png)

[whale_reid_squeezenet_model.zip](https://github.com/OAID/Tengine/files/4342099/whale_reid_squeezenet_model.zip)

**Comment by momandai** [comment]
I think because  fc layer‘s batchnorm is 1d

**Comment by BUG1989** [comment]
@momandai we have fixed the bug #220 , please try it again :)
